Bloon/Features/User Events/WelcomeMatFeature.cs

```diff
@@ -72,8 +72,45 @@ private async Task OnGuildMemberAddedAsync(DiscordClient dClient, GuildMemberAdd
             };
 
             await sbgChannel.SendMessageAsync(string.Empty, embed: userDetails);
-            sbgChannel = await this.dClient.GetChannelAsync(SBGChannels.General);
+        }
+
+        private async Task BloonsideEmbed(DiscordClient dClient, GuildMemberAddEventArgs args)
```

Comment:
Copy-pasta that isn't called from the other method above.

Reply:
not supposed to be called from above. Forgot to add to the event.

Reply:
Are there more stats commands that are going to use this?

Reply:
Nice, still a copy-pasta though.

Reply:
i hand wrote this by short circuiting my usb 2.0 header.

Reply:
Tell me lies, tell me sweet little lies.